.circleci/config.yml

```diff
@@ -578,17 +578,11 @@ jobs:
           store_coverage: false
           pattern: 'ddtracerun'
 
-  test_logging:
-    <<: *contrib_job_small
-    steps:
-      - run_test:
-          pattern: 'test_logging'
-
   stdlib:
-    <<: *contrib_job_small
+    <<: *contrib_job
```

Comment:
I mean, you went from 4 nodes for 4 suites to 1 node for 4 suites to 4 nodes for 5 suites ?

Reply:
lol yes, idk why the logging tests take so long, if it weren't for them then it'd be fine 🤔